### ISSUE TITLE
PusherBroadcaster.php error from null socket

### ISSUE THREAD

**Issue by pursehouse** [laravel/framework#24425]
I get this error from a message submit. The message appears into the chat window, even after refresh, but every message submit returns this http:500 error.

The pusher api configs are all in the .env

```
"message": "",
"exception": "Illuminate\\Broadcasting\\BroadcastException",
"file": "laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Broadcasting\\Broadcasters\\PusherBroadcaster.php",
"line": 116,
"trace": [
    {
        "file": "laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Broadcasting\\BroadcastEvent.php",
        "line": 48,
        "function": "broadcast",
        "class": "Illuminate\\Broadcasting\\Broadcasters\\PusherBroadcaster",
        "type": "->"
    },
    {
        "function": "handle",
        "class": "Illuminate\\Broadcasting\\BroadcastEvent",
        "type": "->"
    },
    {
        "file": "laravel\\vendor\\laravel\\framework\\src\\Illuminate\\Container\\BoundMethod.php",
        "line": 29,
        "function": "call_user_func_array"
    },
```

**Comment by tillkruss** [comment]
Yes, please read the info when opening a new issue.

**Comment by pursehouse** [comment]
The internals break and it doesn't even know why it broke itself. You don't think that's something to be fixed?

**Comment by West1144** [comment]
I have this exact same issue. Any resovle on this?

**Comment by pursehouse** [comment]
@West1144 no, lemme know if you solve it though. Seems odd that the laravel people refuse to even let a ticket exist to fix their own code that breaks with no details of why it broke

**Comment by devcircus** [comment]
New issues come with a template that must be filled in properly before anyone can even begin to help track down and fix possible bugs.

The Laravel team needs to know your Laravel version, PHP version, DB version (if applicable) and a set of steps that can be used to reproduce the bug on a clean install of Laravel. Also, what third part packages are relevant, if any?

Just pasting a stacktrace by itself doesn't help much. The official response was "please read the info when opening a new issue".

**Comment by pursehouse** [comment]
@devcircus  there :)

**Comment by pursehouse** [comment]
@West1144 any luck?

**Comment by emilsrits** [comment]
If you are doing this on http set `'encrypted' => false` in bootstrap.js and config/broadcasting.php

Fixed it for me.

**Comment by OzanKurt** [comment]
@pursehouse Any updates, I am getting the same error.

**Comment by OzanKurt** [comment]
Why are the contributors so careless about some tiny issues?

**Comment by pursehouse** [comment]
@OzanKurt I dunno.. it often seems like if they didn't think of it, they reject it. super annoying
